t/09_parsers.t

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Mojo::Base -strict;
+
+use Mojo::File;
+use Mojo::JSON;
+use Test::More;
+use Test::MockModule;
+use Test::Warnings;
+use Test::MockObject;
+
+use testapi;
+use Parsers::Convert 'yaml2json';
+use Parsers::YAML 'save_yaml_file';
+
+my $valid_yaml = <<'EOM';
+---
+syscalls:
+  ustat01:
+  - message: ustat() is known to fail with EINVAL on Btrfs. Downstream fixes didn't make it to upstream and were removed. bsc#1194208
+    product: opensuse-Tumbleweed
+    retval: ^1$
+EOM
+
+subtest 'yaml' => sub {
+    ok save_yaml_file($valid_yaml, 'valid.yaml'), 'YAML file can be saved with save_yaml_file()';
```

Comment:
For my taste this test wouldn't be sufficient.
It only tests that the function doesn't crash and returns a truthy value.
I would check that the file exists and has the expected content.